easy-ui-tokens/src/font/styles.json

```diff
@@ -80,18 +80,45 @@
         "letter_spacing": {
           "value": "{font.letter_spacing.widen_3.value}"
         },
-        "line_height": { "value": "{font.line_height.1.value}" }
+        "line_height": { "value": "{font.line_height.2.value}" }
       },
       "caption": {
```

Comment:
Keeping caption and overline naming as it's, otherwise we need to update `easypost-web` and `easypost-web-docs`